package.json

```
@@ -154,8 +159,191 @@
     "webpack-cli": "^3.3.11",
     "webpackbar": "^4.0.0"
   },
-  "dependencies": {
-    "chevrotain": "^6.5.0",
-    "tiny-emitter": "^2.1.0"
+  "exports": {
+    ".": {
+      "types": "./typings/index.d.ts",
+      "import": "./es/index.mjs",
+      "require": "./commonjs/index.js"
+    },
+    "./es/i18n/languages": {
+      "types": "./typings/i18n/languages/index.d.ts",
+      "import": "./es/i18n/languages/index.mjs",
+      "require": "./commonjs/i18n/languages/index.js"
+    },
+    "./commonjs/i18n/languages": {
+      "types": "./typings/i18n/languages/index.d.ts",
+      "import": "./es/i18n/languages/index.mjs",
+      "require": "./commonjs/i18n/languages/index.js"
+    },
+    "./es/i18n/languages/csCZ": {
+      "types": "./typings/i18n/languages/csCZ.d.ts",
+      "import": "./es/i18n/languages/csCZ.mjs",
+      "require": "./commonjs/i18n/languages/csCZ.js"
+    },
+    "./commonjs/i18n/languages/csCZ": {
+      "types": "./typings/i18n/languages/csCZ.d.ts",
+      "import": "./es/i18n/languages/csCZ.mjs",
+      "require": "./commonjs/i18n/languages/csCZ.js"
+    },
+    "./es/i18n/languages/daDK": {
+      "types": "./typings/i18n/languages/daDK.d.ts",
+      "import": "./es/i18n/languages/daDK.mjs",
+      "require": "./commonjs/i18n/languages/daDK.js"
+    },
+    "./commonjs/i18n/languages/daDK": {
+      "types": "./typings/i18n/languages/daDK.d.ts",
+      "import": "./es/i18n/languages/daDK.mjs",
+      "require": "./commonjs/i18n/languages/daDK.js"
+    },
```

```suggestion
    "./i18n/languages/csCZ": {
      "types": "./typings/i18n/languages/csCZ.d.ts",
      "import": "./es/i18n/languages/csCZ.mjs",
      "require": "./commonjs/i18n/languages/csCZ.js"
    },
    "./i18n/languages/daDK": {
      "types": "./typings/i18n/languages/daDK.d.ts",
      "import": "./es/i18n/languages/daDK.mjs",
      "require": "./commonjs/i18n/languages/daDK.js"
    },
```
All these language entries should be merged into one so the user can use one path to import the dictionary.

---

Alright, I can add paths `./i18n/languages/xxXX`, but I also want to keep the old paths `./commonjs/...` and `./es/...` for backwards compatibility.

---

done